### PR TITLE
feat(pager): init component

### PR DESCRIPTION
**Pull request by degliwe**
To test:
- Check that "pager" component
- Check the "search page" template is using the "pager" component

### CODE REVIEW

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *ema-styleguide* ready!

Built with commit a8a57a5743bf076d0e843109a7d7c7a889e2ec81

https://deploy-preview-41--ema-styleguide.netlify.com

File: framework/components/ema-pager/ema-pager.twig [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+{% if extra_attributes is defined and extra_attributes is not empty %}
+  {% for attr in extra_attributes %}
+    {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name ~ '="' ~ attr.value ~ '"' %}
+  {% endfor %}
+{% endif %}
+
+{# Print result #}
+<a href="#" class="{{ _css_class }}"{{ _extra_attributes }}>
+Load more
```
Comment (emeryro) [inline framework/components/ema-pager/ema-pager.twig]: href and label should not be hardcoded

File: framework/components/ema-pager/ema-pager.twig [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{#
+  Accepts:
+    - title (string): title of the panel (default: '')
+    - icon (string): name of the ECL-icons class to be used in the heading
```
Comment (emeryro) [inline framework/components/ema-pager/ema-pager.twig]: Seems to be a bad copy/paste

File: framework/components/ema-pager/ema-pager.scss [hunk 3]
```diff
@@ -0,0 +1,21 @@
+/**
+* EMA Separator component
+*/
+
+.ema-pager {
```
Comment (emeryro) [inline framework/components/ema-pager/ema-pager.scss]: default class should have margin: 0;

File: framework/components/ema-pager/ema-pager.scss [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  background-color: map-get($ema-colors, 'light-grey-2');
+  color: map-get($ema-colors, 'grey-3');
+  display: block;
+  font-size: map-get($ecl-font-size, 'xl');
+  line-height: 3;
+  padding: map-get($ecl-spacing, 's');
+  padding: 1rem;
```
Comment (emeryro) [inline framework/components/ema-pager/ema-pager.scss]: duplicated padding

File: framework/components/ema-pager/ema-pager.scss [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+  text-align: center;
+  text-decoration: none;
+  text-transform: uppercase;
+}
+
+.ema-pager:hover {
```
Comment (emeryro) [inline framework/components/ema-pager/ema-pager.scss]: it should probably be the same display on :focus

**Pull request merged**